design/open/commit-metadata/airflow.md

```diff
@@ -0,0 +1,178 @@
+# Metadata for Airflow integration
+
+## About
+
+This is a design to integrate metadata from Airflow into lakeFS.  It
+consists of metadata integration and UI for using it.
+
+## Metadata integration
+
+Metadata that Airflow operators and/or hooks may place on commits to
+associate them with the generating Airflow.
+
+### Stability
```

Comment:
You can always mark it as `beta_` and not worry about compatibilities for now. It will increase our flexibility for the short term for a feature that is on its very early days.